ports/stm/boards/sparkfun_stm32_thing_plus/mpconfigboard.mk

```diff
@@ -0,0 +1,20 @@
+USB_VID = 0x0483
+USB_PID = 0x5740
```

Comment:
Where did these come from?

Reply:
Hi @tannewt. I obtained these values using `lsusb` as soon as I received the board. However, these don't look correct at all :disappointed: 

I will try contacting SparkFun to get the correct values. Thank you for catching this Scott.

Reply:
As a starting point I have posted a question on the SparkFun Forum and will follow this up.

For the time being, I have changed the values to be the same as the MicroMod STM32, just in case anybody is interested in playing with this port.

Reply:
SF is usually pretty responsive with PIDs. Could you post a link to the post here? Generally we don't like to duplicate PIDs beforehand.

Reply:
I understand! The link to my post is below. I posted my question on Friday so it has only been 1-2 business days so far: https://forum.sparkfun.com/viewtopic.php?f=99&t=56759

Reply:
Thanks! Let's give them a week and if we don't hear back then we'll sort something out.

Reply:
It has been just over a week now so I sent an email to `support@sparkfun.com` asking for the PID. I will post any updates here.

Reply:
Apologies for the delay in responding.  The PID for the Thing Plus STM32 is 0x0028.

Reply:
Thanks very much Brian(?). It is appreciated :+1: